talks/abstracts/CollabW2022.md

```diff
@@ -0,0 +1,78 @@
+# SSI Collaborations Workshop 2020 - Mini Workshop
+
+- **Where:** Online! :woman_technologist:
+- **When:** 4th - 7th April 2022
+- **Conference website:** <https://software.ac.uk/cw22>
+
+---
+
+## Name / Email
+
+Sarah Gibson / drsarahlgibson@gmail.com
+
+## Home Institution
+
+The International Interactive Computing Collaboration (2i2c)
+
+## Session Collaborators
+
+_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._
+
+- Meag Doherty, NIH
+- Min Ragan-Kelley, Simula
+- Achintya Rao, The Alan Turing Institute
```

Comment:
If you plan on attending CollabW22, please add your preferred email address.

I do recommend attending as it is a wonderful event, but it is not critical 🙂

Reply:
My Turing address is <arao@turing.ac.uk>. :)

Reply:
```suggestion
## Session Collaborators

_Please include First Name, Surname, Home institution and email address for any additional collaborators on this session proposal, each on a separate line._

- Meag Doherty, NIH
- Min Ragan-Kelley, Simula
- Achintya Rao, The Alan Turing Institute, arao@turing.ac.uk
```